diff -ru orig/crypto/CMakeLists.txt aws-lc-1.55.0/crypto/CMakeLists.txt
--- orig/crypto/CMakeLists.txt	2025-07-17 10:20:13.555869928 -0700
+++ aws-lc-1.55.0/crypto/CMakeLists.txt	2025-07-17 10:41:20.550942735 -0700
@@ -594,7 +594,10 @@
   else()
     add_library(${name} $<TARGET_OBJECTS:crypto_objects> ${CRYPTO_FIPS_OBJECTS} ${module_source})
   endif()
-
+  set_target_properties(${name} PROPERTIES
+    VERSION 4.0.0
+    SOVERSION 4
+    )
   if(FIPS_DELOCATE OR FIPS_SHARED)
     add_dependencies(${name} bcm_o_target)
   endif()
Only in aws-lc-1.55.0/crypto: CMakeLists.txt~
diff -ru orig/ssl/CMakeLists.txt aws-lc-1.55.0/ssl/CMakeLists.txt
--- orig/ssl/CMakeLists.txt	2025-07-17 10:20:13.475870087 -0700
+++ aws-lc-1.55.0/ssl/CMakeLists.txt	2025-07-17 10:41:13.770971337 -0700
@@ -42,7 +42,11 @@
   tls13_client.cc
   tls13_enc.cc
   tls13_server.cc
-)
+  )
+set_target_properties(ssl PROPERTIES
+  VERSION 4.0.0
+  SOVERSION 4
+  )
 target_compile_definitions(ssl PRIVATE BORINGSSL_IMPLEMENTATION)
 
 target_link_libraries(ssl crypto)
Only in aws-lc-1.55.0/ssl: CMakeLists.txt~
