diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
index a2866c770..90d1c7921 100644
--- a/crypto/CMakeLists.txt
+++ b/crypto/CMakeLists.txt
@@ -595,6 +595,11 @@ function(build_libcrypto name module_source)
     add_library(${name} $<TARGET_OBJECTS:crypto_objects> ${CRYPTO_FIPS_OBJECTS} ${module_source})
   endif()
 
+  set_target_properties(${name} PROPERTIES
+    VERSION 3.2.2
+    SOVERSION 3
+    )
+
   if(FIPS_DELOCATE OR FIPS_SHARED)
     add_dependencies(${name} bcm_o_target)
   endif()
diff --git a/ssl/CMakeLists.txt b/ssl/CMakeLists.txt
index 60857fc82..afddcae49 100644
--- a/ssl/CMakeLists.txt
+++ b/ssl/CMakeLists.txt
@@ -43,6 +43,10 @@ add_library(
   tls13_enc.cc
   tls13_server.cc
 )
+set_target_properties(ssl PROPERTIES
+    VERSION 3.2.2
+    SOVERSION 3.2.2
+)
 target_compile_definitions(ssl PRIVATE BORINGSSL_IMPLEMENTATION)
 
 target_link_libraries(ssl crypto)
